.github/workflows/node.js.yml

```diff
@@ -86,61 +42,40 @@ jobs:
 
   # ==== Job: Build and test mangrove-core
   mangrove-core:
-    needs: [security-guard]
-
     runs-on: ubuntu-latest
 
     steps:
 
-    # == Git checkout ==
     - name: Checkout
       uses: actions/checkout@v3
-      # Workaround for https://github.com/npm/cli/issues/2610
       with:
-        persist-credentials: false
-        ref: ${{ env.GIT_REF_TO_TEST }}
         submodules: recursive
 
-    - name: Reconfigure git to use HTTP authentication
```

Comment:
This is not an issue any longer?

Reply:
The linked issue mentions npm, and I don't see any issues with building, so I don't think so? If you know of a concrete issue that does not work in the pipeline even though its green then I'm all ears :D

Reply:
The build was broken due to some dependencies that didn't work in GitHub Actions. If the build works without that workaround I'm all for removing it.